### PR TITLE
#167777756 set up postgres database integration

### PR DESCRIPTION
**Pull request by WinnersProx**
### What does this PR do? 
- setup Postgres database 
#### Description of Task to be completed?
- Download Postgres
- Install Postgres locally
- Configure database in db.config file
- Create migrations file with needed tables
#### How should this be manually tested?
to test this feature insure you have Postgres installed on your machine then run `npm run migrate` in your command-line interface 
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167777756
#### Related Screenshots
db

### CODE REVIEW

File: server/config/db.config.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+const pool = new Pool({ 
+	user: 'me',
+  	host: 'localhost',
+  	database: 'way_farer_db',
+  	password: 'winners1',
+  	port: 5432
+});
+
+export default pool;
```
Comment (oesukam) [inline server/config/db.config.js:11]: Empty line required

File: server/models/db/schema.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+${bookingsTable}
+`;
+
+(async () => {
+    try {
+        await pool.query(queryString);
+    } catch (error) {
+        if (error) console.log(error);
+    }
+})();
```
Comment (oesukam) [inline server/models/db/schema.js:54]: Empty line required

**Review by oesukam — COMMENTED** [review]
@WinnersProx Change your editor configuration to use 2 space instead of 4 for tabulation

File: server/models/db/schema.js (hunk 2 above)
Comment (WinnersProx) [reply server/models/db/schema.js:54]: Okay @oesukam that will be shortly  fixed

**Pull request merged**